index.html

```diff
@@ -52,7 +53,7 @@
             </nav>
             <div class="welcome_container">
                 <h2 class="headline_one">"Hello! Biotech enthusiasts"</h2>
-                <h1 class="headline_two">WORLD BIOTECHNOLOGY<br>CONFERENCE 2023</h1>
+                <h1 class="headline_two">WORLD BIOTECH<br>CONFERENCE 2023</h1>
```

Comment:
- [ ] As this is the main description of your website, it will be better to use a `title` tag, so in this way, it follows the semantic conventions ([title](https://developer.mozilla.org/es/docs/Web/HTML/Element/title#:~:text=El%20elemento%20HTML%20define,dentro%20del%20elemento%20se%20ignoran.&text=Contenido%20de%20metadatos.&text=Texto%20que%20no%20es%20un%20espacio%20en%20blanco%20entre%20elementos.)) of HTML 🟢